src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -459,6 +459,7 @@ private async Task<PackageSpec> GetPackageSpecAsync(ISettings settings)
                         MSBuildStringUtility.IsTrue(GetPropertySafe(_vsProjectAdapter.BuildProperties, ProjectBuildProperties.RestoreLockedMode))),
                     CentralPackageVersionsEnabled = isCpvmEnabled,
                     CentralPackageVersionOverrideDisabled = GetPropertySafe(_vsProjectAdapter.BuildProperties, ProjectBuildProperties.CentralPackageVersionOverrideEnabled).EqualsFalse(),
+                    CentralPackageFloatingVersionsEnabled = MSBuildStringUtility.IsTrue(GetPropertySafe(_vsProjectAdapter.BuildProperties, ProjectBuildProperties.CentralPackageFloatingVersionsEnabled)),
```

Comment:
`GetPropertySafe` uses `GetPropertyValueWithDteFallback`, which should not be used for any new properties.

Please update this so that it calls `projectBuildProperties.GetPropertyValue(string)` instead (the method without the DTE fallbacl)

Reply:
Thanks, done!